packages/element-api/package.json

```diff
@@ -32,10 +33,11 @@
     "swagger-jsdoc": "^3.2.6",
     "swagger-ui-express": "^4.0.2",
     "tar": ">=2.2.2",
+    "websocket": "^1.0.30",
```

Comment:
was this added as a peer deps?

Reply:
Fixed. I added it because it the tests were failing without it but now it seems to work